index.html

```diff
@@ -9,7 +9,23 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
-    <header>
+
+    <section class="nav-open">
+      <div class="close-open">X</div>
+      <nav id="desktop-nav-open">
+        <ul class="ul-open">
+          <li><a class="nav-link-open" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link-open" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link-open" href="#contact"
+              >Contact <i class="far fa-envelope"></i
```

Comment:
- [OPTIONAL] Kindly refactor this block of code to make it more readable. It makes your code looks messy.